filebeat/harvester/reader/multiline.go

```diff
@@ -114,31 +114,34 @@ func (mlr *Multiline) Next() (Message, error) {
 
 func (mlr *Multiline) readFirst() (Message, error) {
 	for {
-		p, err := mlr.reader.Next()
-		if err == nil {
-			if p.Bytes == 0 {
+		message, err := mlr.reader.Next()
+		if err != nil {
+			// no lines buffered -> ignore timeout
+			// TODO: Does this only happen on backoff? Normally is EOF?
```

Comment:
let's add descriptive code comment where `errMultilineTimeout` is defined. Alternative names `sigMultilineTimeout` or `sigReaderTimeout` or `sigBufferTimeout`?